.github/workflows/dispatch_submodule_update.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+name: Dispatch event for cortx submodule
+on:
+  push:
+    branches: [ main ]
+jobs:
+  dispatch:
+    strategy:
+      matrix:
+        repo: ['Seagate/cortx']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
```

Comment:
@venkuppu-chn For security purposes, we need to be forking the third-party actions we are using to control overall exposure instead of using them inline directly.

Reply:
@osowski Did you mean to say that we should fork the third-party action under seagate organization, publish in marketplace and use it. If we do so the line has changed to below. 

uses: seagate/repository-dispatch@v1

Reply:
For the most part, yes. We do NOT need to publish it the Action in Marketplace. We simply need to fork the code to reside underneath the Seagate organization and reference it in the way you mention above. The Actions do not need to be in the marketplace to be consumable, as the `uses` reference is simply a GitHub repository lookup.

Reply:
@osowski As suggested, we have forked and used the fork action in the workflow. Kindly review and let us know.